# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/svgfe-dropshadow.yaml
# Commit: c0f58fd74e56
# Full Hash: c0f58fd74e56b24f67fccae08097d151e21a0f80
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-09 16:15:25
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r 3e2ec3a3b738 -r c0f58fd74e56 gfx/wr/wrench/reftests/filters/svgfe-dropshadow.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/svgfe-dropshadow.yaml	Thu May 09 04:09:00 2024 +0000
@@ -0,0 +1,27 @@
+---
+root:
+  items:
+    - type: stacking-context
+      bounds: [10, 10, 510, 510]
+      filters:
+        - svgfe: SourceGraphic
+        - svgfe: SourceAlpha
+        - svgfe: dropshadow
+          linear: false
+          subregion: [-20, -20, 320, 320]
+          color: red
+          dx: 20
+          dy: 20
+          std_deviation_x: 10
+          std_deviation_y: 10
+          in: 0
+      items:
+        - type: stacking-context
+          bounds: [10, 10, 150, 110]
+          items:
+            - type: rect
+              bounds: [10, 10, 100, 100]
+              color: [127, 127, 127, 1]
+            - type: rect
+              bounds: [50, 30, 100, 60]
+              color: [192, 192, 192, 1]